airflow/opt/providers/etna/etna/metrics/rollup_metrics.py

```diff
@@ -55,4 +55,5 @@ def collect(self) -> Generator[Metric, None, None]:
                 yield metric
 
 def register_rollup_metrics():
-    REGISTRY.register(RollupMetricsCollector())
\ No newline at end of file
+    # REGISTRY.register(RollupMetricsCollector())
+    pass
```

Comment:
Do we lose any metrics by taking this out?

Reply:
Nope, those metrics in the end aren't being used.  I think it's safe to remove these now.